doc/todo/RELEASE

```diff
@@ -9,6 +9,59 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
+
+== Updates ==
+
+- Update doc/COMPILE.md to reflect actually tested setups
+- Write release notes
+
+== Increment Version Number ==
+
+- CMakeLists.txt
+- ~e/scripts/dev/build-debian-package
+- Change VERSION variable in build-server
+- Commit and push the changes
+
+== Check ==
+
+- Everything blue on https://build.libelektra.org/
+- Trigger release pipeline with dry run
+- Inspect the logs, artifacts and the website from the pipeline
+  - [ ] error logs (*.error) must be empty
+  - [ ] check in the (test-*) folder if all tests passed
+  - [ ] check if version is correct in the version file
+  - [ ] 
+
+== Preperation ==
+- Finalize release notes (hash sums etc are already known)
```

Comment:
Do we also move here the release notes to its final name? Or did this already happen earlier?

Reply:
I think moving to final name is a manual step: it's not hard, you need to look at the release notes anyway and there won't be news published on accident by the build system.

Afair the news is not rendered for the website until you add the GUID, but I never checked.

One thing that I did was: I moved the news without GUID to the final place. Then I started the automated release. This way the news is actually in the release tarball and somewhat readable. The GUID, hashes etc. are finalized afterwards, but anyone with the tarball can read the release notes. To me it made sense that you have the changelog in the tarball as well.